website/app/activate/page.tsx

```
@@ -168,12 +177,16 @@ export default function ActivatePage() {
                 <div className="mx-auto grid max-w-4xl gap-6 md:grid-cols-2">
                     {/* Free plan */}
                     <div
+                        role="button"
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)